lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -272,6 +272,9 @@ void SwiftREPL::Initialize() {
   swift.Insert(lldb::eLanguageTypeSwift);
   PluginManager::RegisterPlugin(ConstString("swift"), "The Swift REPL",
                                 &CreateInstance, swift);
+  // The DWARFImporter always claims to be able to load a module, even if it
+  // would fail. Since the REPL doesn't use DWARF type info, we disable it here
```

Comment:
I just discussed this at length with @jimingham. Let's write:
`  // The DWARFImporter always claims to be able to load a module, even if it`
`  // doesn't exist, because it doesn't "import" anything until the first type is requested.`
`  // To make the REPL behave more like the compiler and diagnose when Clang modules`
`  // can't be located up front, the DWARFImporter is disabled in the REPL.`
`  // There are use-cases for loading Clang types from debug info, but this needs a`
`  // better diagnostic mechanism before it can be enabled.`

Reply:
Done!